salt/queues/pgjsonb_queue.py

```diff
@@ -199,29 +210,25 @@ def insert(queue, items):
     with _conn(commit=True) as cur:
         if isinstance(items, dict):
             items = salt.utils.json.dumps(items)
-            cmd = str("""INSERT INTO {0}(data) VALUES('{1}')""").format(
-                queue, items
-            )  # future lint: disable=blacklisted-function
+            cmd = """INSERT INTO {}(data) VALUES('{}')""".format(queue, items)
             log.debug("SQL Query: %s", cmd)
             try:
                 cur.execute(cmd)
```

Comment:
```suggestion
            cmd = """INSERT INTO {}(data) VALUES('%s')""".format(queue)
            log.debug("SQL Query: %s / Params: %s", cmd, items)
            try:
                cur.execute(cmd, [items])
```
To avoid SQL injection.
Please confirm if this actually works, I typed it from memory.